demo/simple-menu.html

```diff
@@ -15,15 +15,17 @@
 <dom-module id="simple-menu">
   <template>
     <style>
-      .content ::content > .iron-selected {
+      ::slotted(*) {
```

Comment:
I think I did this because I thought it would be more consistent with what `simple-menubar` is doing [here](https://github.com/PolymerElements/iron-menu-behavior/pull/78/files#diff-b26b66e5efec6300eaa47f3ef305e41fR18). I can remove it if you think that would be better?